pkg/tfvars/kubevirt/kubevirt.go

```diff
@@ -3,7 +3,6 @@ package kubevirt
 
 import (
 	"encoding/json"
-
 	v1 "github.com/openshift/cluster-api-provider-kubevirt/pkg/apis/kubevirtprovider/v1alpha1"
```

Comment:
nit: this line should remain.

Reply:
Re-added the space between the built-in and kubevirt imported packages.